production/helm/loki/values.yaml

```
@@ -77,8 +77,8 @@ loki:
     allowPrivilegeEscalation: false
   # -- Should enableServiceLinks be enabled. Default to enable
   enableServiceLinks: true
-  # -- Specify an existing secret containing loki configuration. If non-empty, overrides `loki.config`
-  existingSecretForConfig: ""
+  # -- Specify if the configuration is consumed or generated. If true, overrides `loki.config`
```

The way this is worded is very unclear to me, because "true or false" doesn't logically translate to "ConfigMap or Secret" to me.  
```suggestion
  # -- Set to "true" to use an external configuration file (ConfigMap) or to "false" to use a generated secret.
   If true, overrides `loki.config`
```